src/cli/index.js

```diff
@@ -0,0 +1,109 @@
+import qrcode from 'qrcode-terminal';
+import { Promise } from 'es6-promise';
+import chalk from 'chalk';
+import { getInstallations as getBrowserInstallations } from 'testcafe-browser-natives';
+import CliArgumentParser from './argument-parser';
+import createTestCafe from '../';
+
+const ERR_PREFIX = chalk.red('ERROR ');
+
+function exit (code) {
+    // NOTE: give process time to flush output.
+    // It's necessary for some environments.
+    setTimeout(() => process.exit(code), 0);
```

Comment:
You mean I need to bump version in `package.json`? 

> We should increase the timeout to close browsers after the run.

Sounds bad. Seems like it should be done on runner level and not CLI. Ideally, we shouldn't stop the task until all browser `close()` methods not stop working.